src/main/java/com/salesforce/cdp/queryservice/util/TokenHelper.java

```diff
@@ -168,6 +199,76 @@ private static void clearToken(String tokenKey) {
         tokenCache.invalidate(tokenKey);
     }
 
+    private static String un_pw_login(String grantType, String clientId, byte[] clientSecret, String userName, byte[] passwordBytes, String tokenUrl) throws Exception
+    {
+        byte[] grantTypeSegment = ("grant_type=" + grantType).getBytes("utf-8");
+        byte[] clientIdSegment = ("client_id=" + clientId).getBytes("utf-8");
```

Comment:
nit - Constants can be maintained in Constants class

Reply:
I'll move them.